var/spack/repos/builtin/packages/fabulous/package.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
```

Comment:
Optional:  Would prefer the copyright end dat of `2024` in these packages but the checks will allow 2023 for the time being.